buildstockbatch/gcp/gcp.py

```diff
@@ -619,7 +619,7 @@ def start_batch_job(self, batch_info):
             runnables=[bsb_runnable],
             compute_resource=resources,
             # Allow retries, but only when the machine is preempted.
-            max_retry_count=3,
```

Comment:
Was wondering if we should capture anything about which tasks really succeed with a retry of 3 or higher?

Reply:
I think we can get this info from the batch API at the end, if needed.